### PR TITLE
[GCP] Update fields docs and fix inconsistencies after #27974

### PR DESCRIPTION
**Pull request by endorama**
## What does this PR do?

Update the wording in different GCP metricsets about metrics name not being changed from their original GCP counterparts, as they have been updated in https://github.com/elastic/beats/pull/27974 to align them with [Beats naming conventions](https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html).

Remove fields names and description as they are available through `exported fields` section, which is automatically updated from fields definitions and don't risk being outdated.

## Why is it important?

Documentation was confusing and not helping users.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by girodav — APPROVED** [review]

**Review by kaiyan-sheng — APPROVED** [review]

File: x-pack/metricbeat/module/gcp/compute/_meta/docs.asciidoc [hunk 1]
```diff
@@ -1,96 +1,30 @@
...
-
-* `instance.disk.read_ops_count`: Count of disk read IO operations.
-- `instance_name`: The name of the VM instance.
-- `device_name`: The name of the disk device.
-- `storage_type`: The storage type: `pd-standard`, `pd-ssd`, or `local-ssd`.
-- `device_type`: The disk type: `ephemeral` or `permanent`.
-
-* `instance.disk.write_bytes_count`: Count of bytes written to disk
 - `instance_name`: The name of the VM instance.
+    Collected with:
```
Comment (dedemorton) [inline x-pack/metricbeat/module/gcp/compute/_meta/docs.asciidoc:12]: I don't think the layout here is what you expect: https://beats_33006.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/metricbeat-metricset-gcp-compute.html

You probably need a continuation character and some line breaks. Check out our internal docs about list formatting [here](https://github.com/elastic/docs/blob/master/README.asciidoc#lists).

File: x-pack/metricbeat/module/gcp/loadbalancing/_meta/docs.asciidoc [hunk 2]
```diff
@@ -1,34 +1,3 @@
...
-
-[float]
-==== tcp_ssl_proxy
-
-- `loadbalancing.tcp_ssl_proxy.closed_connections`: Number of connections that were terminated over TCP/SSL proxy.
-- `loadbalancing.tcp_ssl_proxy.egress_bytes_count`: Number of bytes sent from VM to client using proxy.
-- `loadbalancing.tcp_ssl_proxy.ingress_bytes_count`: Number of bytes sent from client to VM using proxy.
-- `loadbalancing.tcp_ssl_proxy.new_connections`: Number of connections that were created over TCP/SSL proxy.
-- `loadbalancing.tcp_ssl_proxy.open_connections`: Current number of outstanding connections through the TCP/SSL proxy.
+The `loadbalancing` metricset contains all metrics exported from the https://cloud.google.com/monitoring/api/metrics_gcp#gcp-loadbalancing[Stackdriver API]. The field names are aligned to https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html[Beats naming conventions] with minor modifications to their GCP metrics name counterpart.
```
Comment (dedemorton) [inline x-pack/metricbeat/module/gcp/loadbalancing/_meta/docs.asciidoc:3]:
```suggestion
The `loadbalancing` metricset contains all metrics exported from the https://cloud.google.com/monitoring/api/metrics_gcp#gcp-loadbalancing[Stackdriver API]. The field names are aligned to {beats-devguide}/event-conventions.html[Beats naming conventions] with minor modifications to their GCP metrics name counterpart.
```

**Review by dedemorton — COMMENTED** [review]
Suggested some fixes to the asciidoc tagging. Let me know if you need help with the list formatting. There are some other tricks I know that aren't covered in the readme.

File: x-pack/metricbeat/module/gcp/dataproc/_meta/docs.asciidoc [hunk 3]
```diff
@@ -1,32 +1,5 @@
...
-- `dataproc.cluster.operation.running_count`: Indicates the number of operations that are running on a cluster.
-- `dataproc.cluster.operation.submitted_count`: Indicates the number of operations that have been submitted to a cluster.
-- `dataproc.cluster.yarn.allocated_memory_percentage`: The percentage of YARN memory is allocated.
-- `dataproc.cluster.yarn.apps`: Indicates the number of active YARN applications.
-- `dataproc.cluster.yarn.containers`: Indicates the number of YARN containers.
-- `dataproc.cluster.yarn.memory_size`: Indicates the YARN memory size in GB. Sampled every 60 seconds.
-- `dataproc.cluster.yarn.nodemanagers`: Indicates the number of YARN NodeManagers running inside cluster.
-- `dataproc.cluster.yarn.pending_memory_size`: The current memory request, in GB, that is pending to be fulfilled by the scheduler.
-- `dataproc.cluster.yarn.virtual_cores`: Indicates the number of virtual cores in YARN.
+You can specify a single region to fetch metrics like `us-central1`. Be aware that GCP Storage does not use zones so `us-central1-a` will return nothing. If no region is specified, it will return metrics from all buckets.
```
[the remark this answers is not in the document]
Comment (dedemorton) [reply x-pack/metricbeat/module/gcp/dataproc/_meta/docs.asciidoc:5]: see comment about passive voice being OK later in this review.

File: x-pack/metricbeat/module/gcp/firestore/_meta/docs.asciidoc [hunk 4]
```diff
@@ -1,13 +1,5 @@
...
-You can specify a single region to fetch metrics like `us-central1`. Be aware that GCP Storage does not use zones so `us-central1-a` will return nothing. If no region is specified, it will return metrics from all buckets.
-
-[float]
-=== Metrics
-Here is a list of metrics collected by `firestore` metricset:
-
-- `firestore.document.delete_count`: The number of successful document deletes.
-- `firestore.document.read_count`: The number of successful document reads from queries or lookups.
-- `firestore.document.write_count`: The number of successful document writes.
+You can specify a single region to fetch metrics like `us-central1`. Be aware that GCP Storage does not use zones so `us-central1-a` will return nothing. If no region is specified, it will return metrics from all buckets.
```
Comment (dedemorton) [inline x-pack/metricbeat/module/gcp/firestore/_meta/docs.asciidoc:5]:
```suggestion
You can specify a single region to fetch metrics like `us-central1`. Be aware that GCP Storage does not use zones so `us-central1-a` will return nothing. If no region is specified, metrics are returned from all buckets.
```

File: x-pack/metricbeat/module/gcp/compute/_meta/docs.asciidoc (hunk 1 above)
Comment (endorama) [reply x-pack/metricbeat/module/gcp/compute/_meta/docs.asciidoc]: Thanks, I fixed it in [`7bfe982` (#33006)](https://github.com/elastic/beats/pull/33006/commits/7bfe982f92c47e53e983c975ca4429a595eee40d)

**Review by dedemorton — APPROVED** [review]
LGTM

**Pull request merged**